qiskit_nature/results/electronic_structure_result.py

```diff
@@ -135,24 +135,17 @@ def reverse_dipole_sign(self, value: bool) -> None:
 
     @property
     def total_dipole_moment(self) -> Optional[List[float]]:
-        """Returns total dipole of moment"""
+        """ Returns total dipole of moment """
```

Comment:
Please run `make black` locally to fix the formatting of this file. This should also fix the CI

Reply:
Thanks Max!